storybook/src/docs/app-icon.docs.mdx

```diff
@@ -0,0 +1,135 @@
+import { Meta } from "@storybook/blocks";
+import { IcoFavicon, PngIconAppleTouch, PngIconsWebApp, SvgIconDefault } from "./components/AppIcons";
+
+<Meta title="Docs/Assets/Favicon & App Icons" />
+
+# Favicon & App Icons
+
+Examples of all the assets needed to publish a web app with the proper browser favicon and app names and icons. This set should cover all the combinations of common operating systems (Android, iOS, macOS, Windows) and browsers (Chrome, Safari, Edge, Firefox).
+
+## Overview & examples
+
+### Default SVG Icon
+
+<SvgIconDefault />
+
+```html
+<link rel="icon" type="image/svg+xml" href="icon.svg" />
+```
+
+### Favicon
+
+<IcoFavicon />
+
+```html
+<link rel="icon" href="favicon.ico" sizes="16x16 32x32 64x64" />
+```
+
+### Apple Touch Icon
+
+<PngIconAppleTouch />
+
+```html
+<link rel="apple-touch-icon" href="apple-touch-icon.png" />
+```
+
+### Web App Icons
+
+<PngIconsWebApp />
+
+Include `icons` object in `*.webmanifest` (see [Usage](#usage)):
+
+```json
+{
+  // ...
+  "icons": [
+    { "src": "/icon-192.png", "type": "image/png", "sizes": "192x192" },
+    { "src": "/icon-512.png", "type": "image/png", "sizes": "512x512" }
+  ]
+  // ...
+}
+```
+
+Link the Web Manifest in the `<head>`:
+
+```html
+<link rel="manifest" href="app.webmanifest" />
+```
+
+## Usage
+
+First, install these assets as they come with the [Amsterdam Design System Assets](https://www.npmjs.com/package/@amsterdam/design-system-assets):
+
+```sh
+npm i @amsterdam/design-system-assets
+```
+
+Then you can either use the post install script or manually copy and link the assets.
+
+### (A) Automatic script
+
+Use the [`.postinstall.sh`](https://github.com/Amsterdam/design-system/tree/develop/proprietary/assets/.postinstall.sh) script from the root of your new **Vite React**, **Next.js**, legacy React or generic HTML project to automatically link the assets:
+
+```sh
+# Make sure you are in the root directory of your project!
+sh node_modules/@amsterdam/design-system-assets/.postinstall.sh
```

Comment:
Was a generation tool really necessary? Adding a manifest to your application will also introduce an install button (PWA install) and developers should be well informed on how to do this properly (offline access, caching etc). My question is, is this a part of a design system?

Reply:
It was fun to make :-)

Maybe I could add the option with a CLI flag to opt-in or -out for the Web Manifest part.

Reply:
I think a manifest should be part of the application and not the design system. There are tools like workbox to generate a manifest.

Reply:
The generation of a Web Manifest is now optional.